argcomplete/bash_completion.d/_python-argcomplete

```diff
@@ -208,7 +205,15 @@ _python_argcomplete_global() {
                 _ARGCOMPLETE_SHELL="zsh" \
                 _ARGCOMPLETE_SUPPRESS_SPACE=1 \
                 __python_argcomplete_run "$executable" "${(@)req_argv[1, ${ARGCOMPLETE}-1]}"))
-            _describe "$executable" completions
+            local nosort=()
+            local nospace=()
+            if is-at-least 5.8; then
+                nosort=(-o nosort)
+            fi
+            if [[ "${completions-}" =~ ([^\\]): && "${BASH_REMATCH[2]}" =~ [=/:] ]]; then
```

Comment:
Note this must use `BASH_REMATCH` (as opposed to `match`) because this script uses `setopt local_options BASH_REMATCH`. The indexing is different because we haven't also set `KSH_ARRAYS` and I haven't called `__python_argcomplete_upshift_bash_rematch`.